nodebuilder/gateway/rpc.go

```diff
@@ -0,0 +1,26 @@
+package gateway
+
+import (
+	"github.com/celestiaorg/celestia-node/api/gateway"
+	"github.com/celestiaorg/celestia-node/das"
+	"github.com/celestiaorg/celestia-node/nodebuilder/header"
+	"github.com/celestiaorg/celestia-node/nodebuilder/share"
+	"github.com/celestiaorg/celestia-node/nodebuilder/state"
+)
+
+// Handler constructs a new RPC Handler from the given services.
+func Handler(
+	state state.Module,
+	share share.Module,
+	header header.Module,
+	serv *gateway.Server,
```

Comment:
can you put this param first to distinguish it from the modules pls

Reply:
done